metrics/naming.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package metrics
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+// GetDatastoreKindName gets the full (namespaced) data type name for the given
+// interface (whether a pointer or not).
+func GetDatastoreKindName(data interface{}) string {
+	dataType := reflect.TypeOf(data)
+	for dataType.Kind() == reflect.Ptr {
+		dataType = reflect.Indirect(reflect.ValueOf(
+			data)).Type()
```

Comment:
Did `gofmt` do that? Is it a max column line break rule?

Reply:
We enforce gofmt with a lint-rule, so it's at a minimum gofmt validated. I can try reformat.

Reply:
> We enforce gofmt with a lint-rule

Yes, I know—I was just wondering if `gofmt` made this strange formatting choice. It's odd to read—maybe that's just me? Maybe it's not strange at all?